Manito/Manito/Screens/Detail-Wait/UIComponent/CalendarView.swift

```diff
@@ -0,0 +1,229 @@
+//
+//  CalendarView.swift
+//  Manito
+//
+//  Created by Mingwan Choi on 2022/06/16.
+//
+
+import UIKit
+
+import FSCalendar
+import SnapKit
+
+class CalendarView: UIView {
+    private var selectStartDate = Date()
+
+    let testStartString = "2022-06-20"
+    let testEndString = "2022-06-24"
+
+    private enum CalendarMoveType {
+        case previous
+        case next
+
+        var month: Int {
+            switch self {
+            case .previous:
+                return -1
+            case .next:
+                return 1
+            }
+        }
+    }
+
+    // MARK: - property
+
+    private lazy var previousButton: UIButton = {
+        let button = UIButton()
+        let action = UIAction { _ in
+            self.changeMonth(with: CalendarMoveType.previous)
+        }
+        button.setTitle("<", for: .normal)
+        button.addAction(action, for: .touchUpInside)
+        return button
+    }()
+    private lazy var nextButton: UIButton = {
+        let button = UIButton()
+        let action = UIAction { _ in
+            self.changeMonth(with: CalendarMoveType.next)
+        }
+        button.setTitle(">", for: .normal)
+        button.addAction(action, for: .touchUpInside)
+        return button
+    }()
+    private var calendar: FSCalendar = {
+        let calendar = FSCalendar()
+        calendar.locale = Locale(identifier: "ko_KR")
+        calendar.backgroundColor = .darkGrey001
+        calendar.makeBorderLayer(color: .grey002)
+        calendar.appearance.headerDateFormat = "YYYY년 MM월"
+        calendar.appearance.headerTitleColor = .white
+        calendar.appearance.headerTitleAlignment = .center
+        calendar.appearance.headerMinimumDissolvedAlpha = 0.0
+        calendar.appearance.weekdayTextColor = .grey005
+        calendar.appearance.titleDefaultColor = .white.withAlphaComponent(0.8)
+        // FIXME: weekdayTextColor와 색상이 같아서 수정이 필요해보임
+        calendar.appearance.titlePlaceholderColor = .grey005
+        calendar.appearance.headerTitleFont = .font(.regular, ofSize: 20)
+        calendar.appearance.weekdayFont = .font(.regular, ofSize: 14)
+        calendar.appearance.titleFont = .font(.regular, ofSize: 20)
+        calendar.allowsMultipleSelection = true
+        calendar.appearance.todayColor = .clear
+        return calendar
+    }()
+
+    // MARK: - init
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        render()
+        setupDelegation()
+        setupDateRange()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func render() {
+        self.addSubview(calendar)
+        calendar.snp.makeConstraints {
+            $0.top.leading.trailing.bottom.equalToSuperview()
```

Comment:
p5;

```suggestion
            $0.edges.equalToSuperview()
```
이렇게 쓸 수 있답니다~!

Reply:
오 이건 몰랐네요 !
꿀팁 감삼당 ~~ ^_______^